translations/en.json

```diff
@@ -120,6 +120,16 @@
                 "busy": "Busy"
             }
         },
+        "search_settings": {
+            "headings": {
+                "search_settings_label": "Search"
```

Comment:
the key search.app has already the same value, please use that key instead of creating a new one